packages/theme/src/elevation.ts

```diff
@@ -1,5 +1,12 @@
 export type Elevation = -1 | 0 | 1 | 2 | 3;
 
+export const prevElevation: Record<Elevation, Elevation> = {
```

Comment:
@peter-sanderson is this OK?

Reply:
I think the `useElevation()` hook should be handling this and returning the value :)

Reply:
I think having map for previous the same same way we have for next is generally ok. However it is probably good idea to add the result of it to the hook.

```ts
const { elevation, parentElevation } = useElevation(forcedElevation);
```

Reply:
This is what I meant :—)